source/slang/slang-compiler.cpp

```diff
@@ -1483,17 +1494,36 @@ SlangResult dissassembleDXILUsingDXC(
                     builder << "link ";
                 }
 
+                // 
+                Severity severity = Severity::Error;
+                
                 switch (msg.type)
                 {
-                    case OutputMessage::Type::Error:    builder << "error"; break;
-                    case OutputMessage::Type::Unknown:  builder << "warning"; break;
-                    case OutputMessage::Type::Info:     builder << "info"; break;
+                    case OutputMessage::Type::Unknown:
+                    case OutputMessage::Type::Error:
```

Comment:
Obviously it would be nice if we could just share the same `enum` here, for both `Severity` and `OutputMessage::Type`...